README.md

```diff
@@ -8,158 +8,115 @@ Any idea is :heart:, let discuss about it on [![Join the chat at https://gitter.
 ## Result
 
 <!-- Result from here -->
-Last update: 2018-05-12
+Last update: 2018-06-08
 ```
-OS: Linux (version: 4.16.7-200.fc27.x86_64, arch: x86_64)
-CPU Cores: 4
+OS: Linux (version: 4.16.11-100.fc26.x86_64, arch: x86_64)
+CPU Cores: 8
```

Comment:
Is this the literal number of cores, or is it the number of hyper-threaded cores?  I showed both values in my benchmarker as it is an important value to state and I'd think it should be distinguished here as well.

Reply:
it the number of **physical** cores -> my proc is `AMD FX-8320E Eight-Core Processor`
in fact, I'm running on different computers / kernels